### ISSUE TITLE
H_F with specified HoC or constant CP

### ISSUE THREAD

**Issue by rmcdermo** [firemodels/fds#3214]
1. If you specify a HoC on REAC the "primitive" H_F value for the fuel differs from the "lumped" value of the fuel in the .out file.  It seems like the lumped value is correct.
2. It is often convenient to check CP = 1 kJ/kg/K when debugging.  If you specify this for primitive species you now get HoC=0.  So, then you can set HoC on REAC, but that gives bizarre values for H_F.  First, the values are not negative.  I am sure this is just a reference state issue.  But given that the CP is approximately equal to the variable temperature value, it seems the H_F should be computed to be reasonably close to the default value.

To summarize: (1) definite bug in prim H_F output, (2) double check H_F for specified CP and rescale.

The attached is a simple input file to test all these possibilities.

[H_F_test.txt](https://github.com/firemodels/fds-smv/files/58605/H_F_test.txt)

**Comment by drjfloyd** [comment]
Think I have fixed this.  See if firebot complains overnight

**Comment by drjfloyd** [comment]
I think this remains fixed especially after we also dealt with those temperature issues related to Q_H_CORR.  Randy do you agree?

**Comment by rmcdermo** [comment]
Yes, I imagine this is fine now.
